desktop/NodeManager.ts

```diff
@@ -467,23 +470,28 @@ class NodeManager extends AbstractManager {
           msg: `Can't start the Node: ${spawnError}`,
           stackTrace: '',
         };
-
+        this.nodeLogStream?.write(JSON.stringify(error));
         this.pushToErrorPool({
           type: 'NodeError',
           error,
         });
 
         logger.error('spawnNode', error);
+
+        return;
       }
+
+      this.nodeLogStream?.write(data);
     });
-    this.nodeProcess.stdout?.pipe(logFileStream);
-    this.nodeProcess.stderr?.pipe(logFileStream);
```

Comment:
Not sure we really need it, but I believe it is better to collect both outputs. Is it a problem?

Reply:
Checked. Worked as expected.

Reply:
I think we need to add a second argument to them:
```
    this.nodeProcess.stdout?.pipe(this.nodeLogStream, { end: false });
    this.nodeProcess.stderr?.pipe(this.nodeLogStream, { end: false });
```
Otherwise, we can end up with the same error, having `nodeLogStream` closed via `stdout.pipe`, while `stderr` may still have something to write there... Wdyt?